pkgs/development/libraries/libavif/default.nix

```diff
@@ -22,11 +23,14 @@ stdenv.mkDerivation rec {
 
   # reco: encode libaom slowest but best, decode dav1d fastest
 
+  PKG_CONFIG_GDK_PIXBUF_2_0_GDK_PIXBUF_MODULEDIR = "${placeholder "gdkPixbufLoader"}/${gdk-pixbuf.moduleDir}";
```

Comment:
Environment variables typically go down, but this expression is already a bit lopsided.